internal/resource/config/oauth/client/oauth_client_common.go

```diff
@@ -225,10 +225,15 @@ func readOauthClientResponseCommon(ctx context.Context, r *client.Client, state,
 	// since they can only be configured when the type is set to "OVERRIDE_SERVER_DEFAULT"
 	state.PersistentGrantExpirationType = types.StringPointerValue(r.PersistentGrantExpirationType)
 	if isImportRead && state.PersistentGrantExpirationType.ValueString() == "SERVER_DEFAULT" {
-		state.PersistentGrantExpirationTime = types.Int64Null()
 		state.PersistentGrantExpirationTimeUnit = types.StringNull()
 	} else {
-		state.PersistentGrantExpirationTime = types.Int64PointerValue(r.PersistentGrantExpirationTime)
+		persistentGrantExpirationTime, ok := r.GetPersistentGrantExpirationTimeOk()
```

Comment:
This logic in the else is the same as calling `types.Int64PointerValue` directly.